tests/Unit/Numerical/Spectral/Test_Linearize.cpp

```diff
@@ -0,0 +1,80 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/IndexIterator.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Mesh.hpp"
+#include "DataStructures/StripeIterator.hpp"
+#include "Numerical/Spectral/LegendreGaussLobatto.hpp"
+#include "Numerical/Spectral/Linearize.hpp"
+#include "Utilities/Blas.hpp"
+
+TEST_CASE("Unit.Functors.Linearize", "[Functors][Unit]") {
```

Comment:
No Functors

Reply:
`SPECTRE_TEST_CASE`

@nilsdeppe Relatedly, should Travis help us here by checking for this?